modules/exploits/windows/local/ntusermndragover.rb

```diff
@@ -20,13 +19,15 @@ def initialize(info = {})
     super(update_info(info,
       'Name'           => 'Microsoft Windows NtUserMNDragOver Local Privilege Elevation',
       'Description'    => %q(
-        This module exploits a null pointer deference in win32k triggered by the
-        NTUserMNDragOver syscall.
-        After receiving the menu window object returned by the window procedure function,
-        the xxxMNFindWindowFromPoint function does not effectively check the validity of
-        its member tagPOPUPMENU, causing the subsequent MNGetpItemFromIndex function to
-        trigger the NULL pointer deference.
-        This module has been tested against Windows 7 x86.
+        This module exploits a NULL pointer dereference vulnerability in the win32
```

Comment:
Apply the same fixes I mentioned before here, this needs the update to the first line as per last discussion. Rest of this paragraph looks much better though!